### ISSUE TITLE
When music is not run inside an MPI job it fails with a cryptic message.

### ISSUE THREAD

**Issue by hernando** [INCF/MUSIC#11]
Running music without mpirun (or equivalent) produces this output:

```
music: application_mapper.cc:187: void MUSIC::ApplicationMapper::mapApplications(): Assertion `config_->getDict().size() != 0' failed.
```

As far as I remember, the error message used to be clear. This message is not informative to identify the problem.

**Comment by uahic** [comment]
Ive got the same issue some weeks back, I think it occurs when the number of processes in the music script and the cluster allocation does not match.

**Comment by mdjurfeldt** [comment]
Thanks for your reports. Yes, this is due to some new code lacking proper
error reporting. It is easy to fix and should be fixed.
Den 22 feb 2016 14:56 skrev "Martin Schulze" notifications@github.com:

**Comment by jougs** [comment]
Any news on this? Thanks!

**Comment by hernando** [comment]
Not that I'm aware

**Comment by mdjurfeldt** [comment]
I'll fix this ASAP.

**Comment by mdjurfeldt** [comment]
Fixed in commit 4ce74c9.
